pnpm-workspace.yaml

```diff
@@ -1,6 +1,6 @@
 packages:
   # all packages in subdirs of packages/ and components/
-  - "packages/**"
+  - "packages/*"
```

Comment:
@borisno2 could you please check if you can identify any red flags with this change? Our deployments for docs were failing with the below output, when the packages track all sub directories:

```
Turbo error: Failed to add workspace "" from packages/keystone-6-oauth/providers/github, it already exists at packages/keystone-6-oauth/pages/NextAuthPage
--
Error: Command "cd .. && npx turbo run build --filter={docs}..." exited with 1
```